frontend/src/components/DashboardHeader.vue

```diff
@@ -37,7 +37,7 @@
         </v-tabs>
       </v-expand-transition>
     </v-list>
-  </div>
+  </nav>
```

Comment:
Good intuition! Although this would be the good place to use <header> and then <nav>. BUT! Both these tags are already handled by Vue - you can inspect it using devtools (press F12 on the page to explore its code. Works also on the localhost):

So here we have `<header>` rendered by `v-app-bar`:
![Zrzut ekranu z 2021-07-28 19-20-47](https://user-images.githubusercontent.com/28675024/127373107-1d8f36b2-14b6-40a0-8590-ae45c1734410.png)

And few lines below we have `<nav>` rendered by `v-tabs` already:
![Zrzut ekranu z 2021-07-28 19-21-29](https://user-images.githubusercontent.com/28675024/127373365-14adedc7-0a51-4757-a9e2-20d6edc7b2f4.png)